src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/repository/RecommendationStatusRepository.kt

```diff
@@ -1,13 +1,35 @@
 package uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import org.springframework.data.repository.query.Param
 import org.springframework.stereotype.Repository
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.RecommendationStatusEntity
+import java.time.LocalDateTime
 
 @Repository
 interface RecommendationStatusRepository : JpaRepository<RecommendationStatusEntity, Long> {
   fun findByRecommendationId(@Param("recommendationId") recommendationId: Long): List<RecommendationStatusEntity>
   fun findByRecommendationIdAndName(recommendationId: Long, name: String?): List<RecommendationStatusEntity>
   fun findByName(name: String?): List<RecommendationStatusEntity>
+
+  @Query(
+    value = "SELECT DISTINCT rs.recommendation_id " +
+      "FROM make_recall_decision.public.recommendation_status rs " +
+      "WHERE NOT EXISTS (" +
```

Comment:
It doesn't look like we need 2 sub queries here.  Can the first be replaced with:
`WHERE rs.created < :thresholdDate`?